internal/stringify/fmt_test.go

```diff
@@ -30,7 +30,26 @@ func (a aConcealer) Conceal() string                { return "***" }
 func (a aConcealer) Format(fs fmt.State, verb rune) { io.WriteString(fs, "***") }
 func (a aConcealer) PlainString() string            { return fmt.Sprintf("%v", a.v) }
 
+type testPtrStruct struct {
+	ptrV1 *string
+	inner *testPtrInnerStruct
+}
+
+type testPtrInnerStruct struct {
+	ptrV2 *string
+}
```

Comment:
small nit on naming: `vN` commonly refers to versioning, which isn't the case here.
```suggestion
type aPtrStruct struct {
	val *string
	inner *testPtrInnerStruct
}

type aPtrInnerStruct struct {
	val *string
}
```